tsconfig.json

```diff
@@ -73,7 +73,8 @@
     "package/**/*.ts",
     ".eslintrc.js",
     "jest.config.js",
-    "stories/**.tsx"
+    "stories/**.tsx",
+    "package/src/custom.d.ts"
```

Comment:
remove this cause we are not using this atm.

Reply:
Removed